src/components/FormStepSummary/ComponentValueDisplay.js

```diff
@@ -209,6 +209,16 @@ const CoSignDisplay = ({component, value}) => {
   return <CoSignOld interactive={false} />;
 };
 
+const AddressNLDisplay = ({component, value}) => {
+  if (!value) {
```

Comment:
this won't catch a value like `{postcode: '', houseNumber: ''}` right?

Please add some storybook stories for this (display) component.